src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,26 @@
+package codeu.controller;
+
+import codeu.model.data.Conversation;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Created by CostaA17 on 5/16/18.
+ */
+public class ActivityFeedServlet extends HttpServlet {
```

Comment:
Optional: you might consider adding a test for this file? This is just the prototype so I'll leave that up to you.